cmd/createEmail.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/authorization"
+	"github.com/canonical/identity-platform-admin-ui/internal/config"
+	"github.com/canonical/identity-platform-admin-ui/internal/kratos"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/canonical/identity-platform-admin-ui/internal/mail"
+	"github.com/canonical/identity-platform-admin-ui/internal/monitoring"
+	"github.com/canonical/identity-platform-admin-ui/internal/openfga"
+	"github.com/canonical/identity-platform-admin-ui/internal/pool"
+	"github.com/canonical/identity-platform-admin-ui/internal/tracing"
+	"github.com/canonical/identity-platform-admin-ui/pkg/identities"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/spf13/cobra"
+)
+
+var createEmailCmd = &cobra.Command{
+	Use:   "create-email [identity-id]",
+	Short: "Send an invitation email for an identity",
+	Example: `Send an invitation email:
+
+identity-platform-admin-ui create-email d0bebe60-f4fa-4bdc-b6d6-dff9eb42d287`,
+	Long: `Send an invitation email for an identity if previous email sending efforts failed.`,
+	Args: cobra.ExactArgs(1),
+	Run:  createEmailHandler,
+}
+
+func init() {
+	rootCmd.AddCommand(createEmailCmd)
+}
+
+func createEmailHandler(cmd *cobra.Command, args []string) {
+	ctx := context.Background()
+
+	logger := logging.NewNoopLogger()
+	tracer := tracing.NewNoopTracer()
+	monitor := monitoring.NewNoopMonitor("", logger)
+	wpool := pool.NewWorkerPool(1, tracer, monitor, logger)
+	defer wpool.Stop()
+
+	// Initialize environment variables
+	specs, err := initializeEnv()
+	if err != nil {
+		panic(err)
+	}
+
+	// Initialize identity service
+	service := initializeIdentityService(specs, logger, tracer, monitor, wpool)
+
+	// Get the identity
+	identityID := args[0]
+	identityData, err := service.GetIdentity(ctx, identityID)
+	if err != nil {
+		panic(fmt.Errorf("failed to get the identity: %v", err))
```

Comment:
What is the reason to `panic` in cases like this?

Reply:
yeah, understood your point. Had the same question when writing the code. Just followed the other code to be consistent. We can change it later for tech debt.

Reply:
logger.Fatalf is probably what we want here